src/Prefix.jl

```diff
@@ -448,7 +448,12 @@ function verify(path::AbstractString, hash::AbstractString; verbose::Bool = fals
     # First, it must exist
     if isfile(hash_path)
         # Next, it must contain the same hash as what we're verifying against
-        if readstring(open(hash_path, "r")) == hash
+        cached_hash = ""
```

Comment:
This can be achieved without having to `open` it at all:
```julia
if read(hash_path, String) == hash
```

Reply:
Oh nice.  The wonders of `Compat`.